### PR TITLE
Project Page

### PR DESCRIPTION
**Pull request by 54skyxenon**
Working on the projects page!

### CODE REVIEW

File: src/components/Projects.tsx [hunk 1]
```diff
@@ -2,33 +2,97 @@ import React from 'react';
 import { Helmet } from 'react-helmet';
 import { makeStyles } from '@material-ui/core/styles';
 import { Container, Box, Paper } from '@material-ui/core';
```
Comment (jackblanc) [inline src/components/Projects.tsx]: All imports can go in one line

`import { Card, Container, Box, CardActions, etc } from '@material-ui/core`

File: src/components/Projects.tsx [hunk 2]
```diff
@@ -2,33 +2,97 @@ import React from 'react';
...
+import Typography from '@material-ui/core/Typography';
 
 const useStyles = makeStyles({
   example: {
     color: 'red',
   },
 });
 
-const Template: React.FC = () => {
+const Project: React.FC = () => {
```
Comment (jackblanc) [inline src/components/Projects.tsx]: Projects

File: src/components/Projects.tsx [hunk 3]
```diff
@@ -2,33 +2,97 @@ import React from 'react';
...
   const classes = useStyles();
   return (
     <Container maxWidth="md">
       <Helmet>
-        <title>Write Title Here</title>
+        <title>Code4Community - Projects</title>
         <meta name="description" content="Write a description for SEO here" />
```
Comment (jackblanc) [inline src/components/Projects.tsx]: Please write an SEO message

File: src/components/Projects.tsx [hunk 4]
```diff
@@ -2,33 +2,97 @@ import React from 'react';
...
       <Box my={4}>
         <Paper className={classes.example} />
+
+        <Card className="project-card">
+          <CardActionArea>
+            <CardMedia
+              component="img"
+              className="project-img"
+              image="./llb.png"
+              title="LLB"
```
Comment (jackblanc) [inline src/components/Projects.tsx]: Getting an ESLint warning for this one
Screen Shot 2020-08-02 at 8 35 38 AM
The alt property is required on the internal image I believe.

File: src/components/Projects.tsx [hunk 5]
```diff
@@ -2,33 +2,97 @@ import React from 'react';
...
+                The Sajni Center is a space of hope and healing for children with life-threatening illness or chronic 
+                medical conditions and their families. Here they host events for the children and their families. We created 
+                the event planning and ticket purchasing system for all the events hosted by Lucy's Love Bus.
+              </Typography>
+            </CardContent>
+          </CardActionArea>
+          <CardActions>
+            <Button size="small" color="primary">
+              Share
+        </Button>
```
Comment (jackblanc) [inline src/components/Projects.tsx]: Is this file formatted?

**Review by jackblanc — COMMENTED** [review]
Great start here! Few things on file organization: 
1. Create a directory `containers` and move your `Projects.tsx` file into it
2. Create a component `ProjectCard` (or a better name if you have one) in `/components`. This component should contain the card you wrote (lines 30-60), then we'll use our component to avoid duplicated code. The props should be a title, body text, img. 

Notes on design: 
1. The card seems to be clickable as a whole, but it doesn't do anything when you click, which is an odd UX to me
2. We need spacing between cards
3. Each page should have a header with a title and what the page is about (would be good to separate into a component)

Okay I'm done now. Great work! Lots of comments but you're going the right direction 👍

File: src/components/Projects.tsx [hunk 6]
```diff
@@ -2,33 +2,97 @@ import React from 'react';
...
     <Container maxWidth="md">
       <Helmet>
-        <title>Write Title Here</title>
+        <title>Code4Community - Projects</title>
         <meta name="description" content="Write a description for SEO here" />
       </Helmet>
       <Box my={4}>
         <Paper className={classes.example} />
+
+        <Card className="project-card">
```
Comment (rymaju) [inline src/components/Projects.tsx]: Not really sure what this is, should be {classes.cssClass} like on line 28.

File: src/components/Projects.tsx [hunk 7]
```diff
@@ -2,33 +2,97 @@ import React from 'react';
...
   const classes = useStyles();
   return (
     <Container maxWidth="md">
       <Helmet>
-        <title>Write Title Here</title>
+        <title>Code4Community - Projects</title>
         <meta name="description" content="Write a description for SEO here" />
       </Helmet>
       <Box my={4}>
         <Paper className={classes.example} />
```
Comment (rymaju) [inline src/components/Projects.tsx]: Should this still be in here?

File: src/components/Projects.tsx [hunk 8]
```diff
@@ -2,33 +2,97 @@ import React from 'react';
...
+        <Card className="project-card">
+          <CardActionArea>
+            <CardMedia
+              component="img"
+              className="project-img"
+              image="./llb.png"
+              title="LLB"
+            />
+            <CardContent>
+              <Typography gutterBottom variant="h5" component="h2">
```
Comment (rymaju) [inline src/components/Projects.tsx]: Wondering why we need component h2? Looks familar though, is this from the documentation?

File: src/components/Projects.tsx [hunk 9]
```diff
@@ -2,33 +2,97 @@ import React from 'react';
...
+                Integrative therapies ease children’s pain and anxiety during and after traditional medical treatments.
+                The Sajni Center is a space of hope and healing for children with life-threatening illness or chronic 
+                medical conditions and their families. Here they host events for the children and their families. We created 
+                the event planning and ticket purchasing system for all the events hosted by Lucy's Love Bus.
+              </Typography>
+            </CardContent>
+          </CardActionArea>
+          <CardActions>
+            <Button size="small" color="primary">
+              Share
```
Comment (rymaju) [inline src/components/Projects.tsx]: Where would this button go eventually? Maybe we want to have a list of buttons to share on different platforms: just a thought.

File: src/components/Projects.tsx [hunk 10]
```diff
@@ -2,33 +2,97 @@ import React from 'react';
...
+              Learn More
+        </Button>
+          </CardActions>
+        </Card>
+
+        <Card className="project-card">
+          <CardActionArea>
+            <CardMedia
+              component="img"
+              className="project-img"
```
Comment (rymaju) [inline src/components/Projects.tsx]: Same stuff as before and on line 62, the classes aren't correct. Should be className={classes.cssClass}.

**Review by rymaju — DISMISSED** [review]
Mostly agree with everything Jack said, I think that we want to have more content on the Projects page in general and think through the interactions on the page. For example, where does clicking on the card body take you? Where does "Share" take you? Where does "Learn More" take you? Are we going to have individual pages for projects or only cards?

Also, we should consider having a header/title on the page as well as some spacing between the cards. Other than that the idea is generally correct - just needs a lot of polishing and cleaning up. We can talk more about what exactly the vision for this page is during Monday's meeting.

Lets just fix up some of the css class things, delete that leftover paper tag, run `npm run fix` to lint, rename a couple of things, add a description tag, and we should be good to merge.

File: src/components/Projects.tsx (hunk 1 above)
Comment (54skyxenon) [reply src/components/Projects.tsx]: fixed

File: src/components/Projects.tsx (hunk 2 above)
Comment (54skyxenon) [reply src/components/Projects.tsx]: fixed

File: src/components/Projects.tsx (hunk 3 above)
Comment (54skyxenon) [reply src/components/Projects.tsx]: done

File: src/components/Projects.tsx (hunk 4 above)
Comment (54skyxenon) [reply src/components/Projects.tsx]: strange, doesn't appear for me, but I fixed it anyways

File: src/components/Projects.tsx (hunk 5 above)
Comment (54skyxenon) [reply src/components/Projects.tsx]: yeah, it's kind of weird

File: src/components/Projects.tsx (hunk 7 above)
Comment (54skyxenon) [reply src/components/Projects.tsx]: why not? it's just the title

File: src/components/Projects.tsx (hunk 8 above)
Comment (54skyxenon) [reply src/components/Projects.tsx]: ctrl-c ctrl-v :)

File: src/components/Projects.tsx (hunk 9 above)
Comment (54skyxenon) [reply src/components/Projects.tsx]: I like the way it is right now

File: src/components/Projects.tsx (hunk 10 above)
Comment (54skyxenon) [reply src/components/Projects.tsx]: fixed

File: src/components/Projects.tsx (hunk 8 above)
Comment (54skyxenon) [reply src/components/Projects.tsx]: it looked nice the way it was so I left it as that

File: src/components/Projects.tsx (hunk 6 above)
Comment (54skyxenon) [reply src/components/Projects.tsx]: fixed

**Comment by 54skyxenon** [comment]
Do you have ideas on where you want the buttons to lead? @jackblanc @rymaju 
As of right now, I fixed the trivial PR comments you guys left.

**Comment by 54skyxenon** [comment]
All of the high-level suggestions have been addressed in some way or another as of now. I think it's ready for deployment!

**Review by rymaju — COMMENTED** [review]
Leaving a comment so maybe aws will make a web deploy for you

**Comment by jackblanc** [comment]
Amplify link: https://bliang-projects-page.d2ftpg0u6tekg0.amplifyapp.com/

File: src/App.tsx [hunk 11]
```diff
@@ -8,7 +8,7 @@ import ApplyDeveloper from './components/ApplyDeveloper';
 import ApplyProductDesigner from './components/ApplyProductDesigner';
 import ApplyBrandDesigner from './components/ApplyBrandDesigner';
 import Partner from './components/Partner';
-import Projects from './components/Projects';
+import Projects from './components/containers/Projects';
```
Comment (jackblanc) [inline src/App.tsx]: @rymaju do we want this directory structure?

File: src/components/containers/ProjectCard.tsx [hunk 12]
```diff
@@ -0,0 +1,75 @@
...
+
+import {
+    CardContent,
+    Typography,
+    CardMedia,
+    Card,
+    makeStyles,
+    Box,
+    Grid
+}
```
Comment (jackblanc) [inline src/components/containers/ProjectCard.tsx]: It looks like this file hasn't been linted

**Review by jackblanc — CHANGES_REQUESTED** [review]
Why do we have images in the public folder? Can we add them to S3?

File: src/components/containers/ProjectCard.tsx (hunk 12 above)
Comment (54skyxenon) [reply src/components/containers/ProjectCard.tsx]: I made sure to lint it

File: src/App.tsx (hunk 11 above)
Comment (54skyxenon) [reply src/App.tsx]: Changed

**Review by jackblanc — APPROVED** [review]

**Review by rymaju — APPROVED** [review]
mostly looks good to me. This merge is long past due, but I want to come back later to look at the styling. It could look better.

**Pull request merged**